packages/ssz/src/type/byteArray.ts

```diff
@@ -149,3 +159,16 @@ export abstract class ByteArrayType extends CompositeType<ByteArray, ByteArray,
 
   protected abstract assertValidSize(size: number): void;
 }
+
+export function getBlocksBytes(value: Uint8Array, blocksBuffer: Uint8Array): Uint8Array {
+  if (value.length > blocksBuffer.length) {
+    throw new Error(`data length ${value.length} exceeds blocksBuffer length ${blocksBuffer.length}`);
+  }
+
+  blocksBuffer.set(value);
+  const valueLen = value.length;
+  const blockByteLen = Math.ceil(valueLen / 64) * 64;
```

Comment:
It might be helpful to break out a helper function so its clear why this is happening everywhere.

```ts
export function getPaddedByte32Count(buf: ArrayBuffer): number {
    return Math.ceil(buf.length / 32);
}

export function getPaddedByte64Count(buf: ArrayBuffer): number {
    return Math.ceil(buf.length / 64);
}
```